packages/fetchai/agents/erc1155_deployer/README.md

```diff
@@ -0,0 +1,11 @@
+# ERC1155 Deployer
+
+This is an agent that sells data via a smart contract.
+
+## Description
+
+This agent uses its primary skill, the `fetchai/erc1155_deploy` skill, to register its 'data-selling' service on the `SOEF`. It can then be contacted by another agent (for example the `fetchai/generic_buyer` agent) to provide specific data. Once such aa request is made, this agent negotiates the terms of a deal using the `fetchai/fipa` protocol, and if an agreement is reached, it delivers the data after receiving payment via its deployed smart contract.
```

Comment:
```suggestion
This agent uses its primary skill, the `fetchai/erc1155_deploy` skill, to register its 'data-selling' service on the `SOEF`. It can then be contacted by another agent (for example the `fetchai/erc1155_client` agent) to provide specific data. Once such aa request is made, this agent negotiates the terms of a deal using the `fetchai/fipa` protocol, and if an agreement is reached, it delivers the data after receiving payment via its deployed smart contract.
```

Reply:
The key is that this agent demonstrates deployment of an erc1155 smart contract and trading using it. The negotiation is not full fipa (it uses it though). The focus is to demo how to use a smart contract.

Reply:
Done